src/Workleap.DomainEventPropagation.Subscription.PullDelivery/Workleap.DomainEventPropagation.Subscription.PullDelivery.csproj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <IsPackable>true</IsPackable>
+    <IncludeSymbols>true</IncludeSymbols>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>../Workleap.DomainEventPropagation.snk</AssemblyOriginatorKeyFile>
+    <RootNamespace>Workleap.DomainEventPropagation</RootNamespace>
+    <!-- This .NET error has been fixed in .NET 7 -->
+    <NoWarn>$(NoWarn);AD0001</NoWarn>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.7.0" />
```

Comment:
This package has a known vulnerability with a high score; it has been fixed from versions `1.10.2` and forward based on the security advisory.

Reply:
This was fixed in #118